doc/Test common standards.md

```diff
@@ -150,10 +151,20 @@ describe('User', () => {
 
 The `[Function or acceptance criterion]` component is `calling TryChangeName` as this is the method name of the `User` unit, the `[Action or Parameters]` component is `wrong password` as a description that a wrong argument is specified, the `[DesiredResult]` component is `return false`, because the test condition is that the call to the subjects function return a value of false to indicate failure.
 
+## Running Tests
```

Comment:
There is a TODO for this above
> **TODO**: Add some short description of how to execute the tests and give a link for further information and instructions.  

Wither put this section there or leave it here, but the TODO is resolved and can be removed from the document